src/sass/_progressive_profiling_component.scss

```diff
@@ -0,0 +1,26 @@
+.pp-page--login-button-width{
```

Comment:
can we rename the file to `_progressive_profiling_page.scss`? This suggestion applies to login scss file as well.